vita3k/gui/src/settings_dialog.cpp

```diff
@@ -1119,25 +1119,27 @@ void draw_settings_dialog(GuiState &gui, EmuEnvState &emuenv) {
 
         // Tracy modules list
         if (ImGui::BeginListBox(tracy_modules_list_label.c_str(), { 0.0f, ImGui::GetTextLineHeightWithSpacing() * 8.25f + ImGui::GetStyle().FramePadding.y * 2.0f })) {
+            // Get all HLE modules available for advanced profiling using Tracy. Do it only once.
+            static std::vector<std::string> tracy_modules{};
+            if (tracy_modules.empty())
+                tracy_module_utils::get_available_module_names(tracy_modules);
```

Comment:
```suggestion
            static std::vector<std::string> tracy_modules = tracy_module_utils::get_available_module_names(tracy_modules);
```
Will be called only once